content/en/Platform Deep Dive/Scans/_index.md

```diff
@@ -24,16 +24,15 @@ We use the following IP Address to make requests to your target: **52.19.40.38**
 
 <iframe src="https://play.vidyard.com/t2kwGzh5zhfgP5rupyKrrB" width="640" height="360" frameborder="0" allowfullscreen></iframe>
 
-
 ### Add a Target
 
 - To add a [target](/platform-deep-dive/scans/targets), select the Add Target button
 
-![Create Target](/deepdive/scans/1_CreateTarget.png "Create Target")<br>
+{{% image src="/deepdive/scans/1_CreateTarget.png" alt="Create Target" %}}
 
 - Add a name for the target and the URL
 
-![Add Target](/deepdive/scans/2_AddTarget.png "Add Target")<br>
+{{% image src="/deepdive/scans/2_AddTarget.png" alt="Add Target" %}}
 
 See more about how to configure a target in [configuring a target]
```

Comment:
Additionally, or in place of, I think we should link to https://deploy-preview-495--cobalt-docs.netlify.app/platform-deep-dive/scans/targets/#adding-a-web-or-api-target because the image above shows those options but we don't even mention how to handle creating Web vs API targets